src/layout/Hero.scss

```diff
@@ -2,20 +2,71 @@
 @import '~@patternfly/patternfly/sass-utilities/_all';
 
 .ins-c-hero {
+    --ins-c-hero--MinHeight: 0;
+    --pf-c-page__main-section--PaddingTop: var(--pf-global--spacer--2xl);
+    --pf-c-page__main-section--PaddingBottom: var(--pf-global--spacer--2xl);
 
-    background: url("/apps/chrome/assets/images/fedora-background-left.svg");
-    background-repeat: no-repeat;
-    background-size: cover;
+    --ins-c-hero--before--Width: 0;
+    --ins-c-hero--before--Opacity: 0;
+    --ins-c-hero--before--BackgroundSize: 140px;
 
-    h1.ins-c-hero__title { color: white; }
+    --ins-c-hero--before--md--Width: 38%;
+    --ins-c-hero--before--md--Opacity: 1;
+    --ins-c-hero--before--md--BackgroundSize: 234px;
 
-    h2.ins-c-hero__sub-title { color: var(--pf-global--Color--200); }
+    --ins-c-hero--before--lg--BackgroundSize: 242px;
 
-    @media screen and (min-width: $pf-global--breakpoint--xs) {
-        @include rem('min-height', 400px);
+    position: relative;
+
+    &::before {
+        position: absolute;
+        right: 0;
+        top: 0;
+        bottom: 0;
+        width: var(--ins-c-hero--before--Width);
+        content: '';
+        opacity: var(--ins-c-hero--before--Opacity);
+        background-image: #{url('https://raw.githubusercontent.com/RedHatInsights/frontend-assets/0331b39e1a370e5f1c243452330df61f4e1b57bc/src/background-images/bg__hero.svg')};
```

Comment:
You don't have to use `githubusercontent` as URL, just `https://cloud.redhat.com/apps/frontend-assets/src/background-images/bg__hero.svg` is the URL correct @ryelo ?

Reply:
Correct